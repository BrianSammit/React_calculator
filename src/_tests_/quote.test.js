import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import '@testing-library/react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quotes';

describe('Home', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Router>
          <Quote />
        </Router>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
