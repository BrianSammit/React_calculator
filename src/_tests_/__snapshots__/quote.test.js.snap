// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div
  className="quote"
>
  <h2>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul ThurstonHomeCalculatorQuote
  </h2>
</div>
`;
